libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -62,8 +62,8 @@ function branchInit(header, key) {
 export default async function init(el) {
   const header = document.querySelector('.global-navigation');
   if (!header) return;
-  const row = el.querySelector(':scope > div');
-  const product = row.textContent.trim().toLowerCase();
+  const classListArray = Array.from(el.classList);
+  const product = classListArray.find((token) => token.startsWith('product-')).split('-')[1];
```

Comment:
I'm personally not a huge fan of mixing code changes with unit tests - since it requires regression testing - rather than saying "Those are only unit tests, it's zero-impact and doesn't need any testing since at worst we have broken tests (and not a CSO)"

It's completely fine to do, but that's my rationale behind it